server/db/models/User.ts

```diff
@@ -11,21 +11,16 @@ const UserFactory = (
   DataTypes: Sequelize.DataTypes,
 ): Sequelize.Model<UserInstance, UserAttributes> => {
   const attributes: SequelizeAttributes<UserAttributes> = {
-    id: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
-    firstName: {
-      type: DataTypes.STRING,
-    },
-    lastName: {
-      type: DataTypes.STRING,
-    },
+    firstName: { type: DataTypes.STRING },
+    lastName: { type: DataTypes.STRING },
     email: {
       type: DataTypes.STRING(255),
       allowNull: false,
       unique: true,
     },
     password: {
       type: DataTypes.STRING(255),
-      allowNull: false,
+      allowNull: true,
```

Comment:
when creating a user, passwords should be null. When we send an email and the user verifies their account, that will be where they create a new password.

Reply:
Maybe comment it if we're worried about forgetting this:
```suggestion
      allowNull: true, // create password upon verification
```